liquibase-standard/src/test/groovy/liquibase/parser/core/xml/XMLChangeLogSAXParserTest.groovy

```diff
@@ -142,7 +142,7 @@ class XMLChangeLogSAXParserTest extends Specification {
         then:
         def e = thrown(ChangeLogParseException)
         e.message.contains("Error parsing line")
-        e.message.contains("iDontKnowWhatImDoing")
```

Comment:
I just noticed that: this error should not have been changed. in this test, `databaseChangeLog` IS present. But this second tag (`<iDontKnowWhatImDoing />` - line 49) does not exist. Looks like this PR is showing this error message for any invalid XML errors.

Reply:
Test change reverted